packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/visits-table/visits-table.component.tsx

```diff
@@ -103,6 +104,37 @@ const VisitTable: React.FC<VisitTableProps> = ({ showAllEncounters, visits }) =>
     setFilter(selectedItem);
   };
 
+  const handleDeleteEncounter = React.useCallback(
+    (encounterUuid: string, encounterTypeName?: string) => {
+      const close = showModal('delete-encounter-modal', {
+        close: () => close(),
+        encounterTypeName: encounterTypeName || '',
+        onConfirmation: () => {
+          const abortController = new AbortController();
+          deleteEncounter(encounterUuid, abortController)
+            .then(() => {
+              showToast({
+                title: t('deleteEncounter', 'Delete Encounter', { encounter: encounterTypeName }),
+                description: `Encounter ${t('successfullyDeleted', 'successfully deleted')}`,
+                kind: 'success',
+              });
+
+              setFilteredRows(visits?.filter((encounter) => encounter.id !== encounterUuid));
```

Comment:
Instead of this, I think we can call the mutate function of the SWR hook calling in the Encounters.

Reply:
Hi @vasharma05, Thanks for the feedback.

This component receives the rows by prop.
I tried several approaches with mutate but without success.

For example, I tried: 
```mutate(`/ws/rest/v1/encounter/${encounterUuid}`); ```
and
```mutate(`${fhirBaseUrl}/Encounter/${encounterUuid}`);```

Am I doing something wrong?
Thanks,